### PR TITLE
added a no-scroll feature to the body element when the nav is active …

### PR DESCRIPTION
**Pull request by Nnadivictory25**
added a no-scroll feature to the body element when the nav is active on mobile to prevent users from being able to scroll while the nav is active :)

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -19,11 +20,15 @@ menuBtn.addEventListener('click', () => {
         menuView.classList.remove('slide-out')
         menuView.classList.add('slide-in');
         dim.style.display = 'block';
+        body.classList.remove('no-scroll')
```
Comment (NARUDESIGNS) [inline app.js:23]: You added the class list here and removed it in the exact next line.
I'm curious how it works, can you share a short screen record of your solution?

File: app.js [hunk 2]
```diff
@@ -19,11 +20,15 @@ menuBtn.addEventListener('click', () => {
...
+        body.classList.add('no-scroll')
         state = !state;
     }
     else {
         menuView.classList.add('slide-out');
         menuView.classList.remove('slide-in');
+        body.classList.add('no-scroll')
+        body.classList.remove('no-scroll')
```
Comment (NARUDESIGNS) [inline app.js:29]: You added the class list here on `line 30` and removed it in the exact next line on `line 31`
I see something similar on `line 23` and `line 24`
I'm curious how it works, can you share a short screen record of your solution?

Comment (Nnadivictory25) [reply app.js:29]: Made a mistake 
Even tho it works , `line 30` is irrevelant 
Corrected it now


https://user-images.githubusercontent.com/71198309/194110202-a62a0a80-c978-4208-ab26-82d3e73da33b.mp4

Comment (Nnadivictory25) [reply app.js:29]: `line 23` also is not relevant 
removed also 

pardon my ignorance boss :)

**Comment by NARUDESIGNS** [comment]
Thanks for your contribution!
@Nnadivictory25

**Comment by Nnadivictory25** [comment]
My pleasure !

**Pull request merged**